dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -623,13 +623,33 @@ private void updateTool(List<String> args) {
                 final String dockerfilePath = optVal(args, "--dockerfile-path", container.getDefaultDockerfilePath());
                 final String toolname = optVal(args, "--toolname", container.getToolname());
                 final String gitUrl = optVal(args, "--git-url", container.getGitUrl());
+                final String defaultTag = optVal(args, "--default-version", container.getDefaultVersion());
 
                 container.setDefaultCwlPath(cwlPath);
                 container.setDefaultWdlPath(wdlPath);
                 container.setDefaultDockerfilePath(dockerfilePath);
                 container.setToolname(toolname);
                 container.setGitUrl(gitUrl);
 
+                // if valid version
+                boolean updateVersionSuccess = false;
+
+                for (Tag tag : container.getTags()) {
+                    if (tag.getName().equals(defaultTag)) {
+                        container.setDefaultVersion(defaultTag);
+                        updateVersionSuccess = true;
+                        break;
+                    }
+                }
+
+                if (!updateVersionSuccess && defaultTag != null) {
+                    out("Not a valid tag.");
+                    out("Valid tags include:");
+                    for (Tag tag : container.getTags()) {
+                        out(tag.getReference());
+                    }
```

Comment:
I think this might need to abort here, otherwise this will exit with exit code 0